iwyu_location_util.h

```diff
@@ -37,8 +37,8 @@
 // file path, line number, column number, and which file (if any)
 // #includes the file.
 
-#ifndef DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_IWYU_FILE_UTIL_H_
-#define DEVTOOLS_MAINTENANCE_INCLUDE_WHAT_YOU_USE_IWYU_FILE_UTIL_H_
+#ifndef INCLUDE_WHAT_YOU_USE_IWYU_LOCATION_UTIL_H_
+#define INCLUDE_WHAT_YOU_USE_IWYU_LOCATION_UTIL_H_
```

Comment:
`git push -f ...` should override this (`-f` for 'force'). But your error message is different from what I'd expect, so maybe your situation or git version is different.